rosdep/python.yaml

```diff
@@ -8670,6 +8680,16 @@ uavcan-pip:
   ubuntu:
     pip:
       packages: [uavcan]
+univorn-pip:
+  debian:
+    pip:
+      packages: [uvicorn]
+  fedora:
+    pip:
+      packages: [univorn]
+  ubuntu:
+    pip:
+      packages: [uvicorn]
```

Comment:
It seems there is a mix of `univorn` and `uvicorn`, is this intended?

Reply:
Good catch!